charts/datadog/templates/_container-host-release-volumemounts.yaml

```diff
@@ -1,4 +1,5 @@
 {{- define "linux-container-host-release-volumemounts" -}}
+{{- if (not .Values.datadog.processAgent.disableOsReleaseFileMount) }}
```

Comment:
the option `datadog.systemProbe.enableDefaultOsReleasePaths` (with `true` as default value already exist. 
Can reuse this option? Or maybe create a more global option such as `datadog.enableDefaultOsReleasePaths` and deprecate `datadog.systemProbe.enableDefaultOsReleasePaths`?

Reply:
updated :)